lib/china_regions/helpers/form_helper.rb

```diff
@@ -4,9 +4,9 @@ module Helpers
     module FormHelper
       def region_select(object_name, methods, options = {}, html_options = {})
         output = ''
-        preselected_choices = {}
+        @preselected_choices = {}
```

Comment:
I had done this to cut down the amount of arguments for the `get_choices` method and the `prioritize_choices` method.